### PR TITLE
V10.2.1

### PR DESCRIPTION
**Pull request by mcardosos**
PTAL
@marstr @jhendrixMSFT

### CODE REVIEW

File: arm/mysql/models.go [hunk 1]
```diff
@@ -232,31 +316,108 @@ type ServerProperties struct {
 	FullyQualifiedDomainName *string            `json:"fullyQualifiedDomainName,omitempty"`
 }
 
+// IServerPropertiesForCreate is interface used for polymorphic Properties in ServerForCreate
+type IServerPropertiesForCreate interface {
```
Comment (jhendrixMSFT) [inline arm/mysql/models.go]: The 'I' prefix is a C#-ism, I think we should drop it and simply call the type ServerPropertiesForCreate.

Comment (mcardosos) [inline arm/mysql/models.go]: Fixed

File: arm/postgresql/models.go [hunk 2]
```diff
@@ -210,10 +223,81 @@ type Server struct {
...
+		if err != nil {
+			return err
+		}
+		sfc.Tags = &tags
+	}
+
+	return nil
+}
+
+func unmarshalIServerPropertiesForCreate(b []byte) (ServerPropertiesForCreate, error) {
```
Comment (marstr) [inline arm/postgresql/models.go]: Shouldn't this be renamed to not include the "I" in the SerePropertiesForCreate?

Comment (mcardosos) [inline arm/postgresql/models.go]: Fixed

**Pull request merged**